exporter/signalfxexporter/internal/translation/logdata_to_signalfxv2.go

```diff
@@ -48,18 +48,8 @@ func LogRecordSliceToSignalFxV2(
 func convertLogRecord(lr pdata.LogRecord, resourceAttrs pdata.AttributeMap, logger *zap.Logger) (*sfxpb.Event, bool) {
 	attrs := lr.Attributes()
 
-	categoryVal, ok := attrs.Get(splunk.SFxEventCategoryKey)
-	if !ok {
-		return nil, false
-	}
```

Comment:
Not sure why this should be moved below? It maybe still good to drop unrelated logs right away and avoid further manipulations?

Reply:
Thanks for the catch. I didn't realize this was intentionally a different path that needs to drop unrelated logs. Will change back.